google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -97,10 +97,13 @@ def get_group_members(self, group_key):
         request = members_stub.list(groupKey=group_key,
                                     maxResults=FLAGS.max_results_admin_api)
 
-        results = self._build_paged_result(
+        paged_results = self._build_paged_result(
             request, members_stub, self.rate_limiter)
 
-        return [item.get('members', []) for item in results]
+        group_members = []
```

Comment:
If you're up for fixing this once, if not just open an issue, but we should write another helper method to un-bundle the results of a paged request.

It'd accept just one argument, the string of the key and do what you're doing on lines 103-106